pyiron_atomistics/sphinx/base.py

```diff
@@ -117,7 +117,11 @@ def __getitem__(self, item):
                     result = super().__getitem__(tag)
                 else:
                     result = result[tag]
-                if hasattr(result, "list_nodes") and "TYPE" in result.list_nodes():
+                if (
+                    hasattr(result, "list_nodes")
```

Comment:
```suggestion
                    isinstance(result, HasGroups)
```
you might have to import `HasGroups` from pyiron_base, though.